### PR TITLE
Add microsoft-azure-storage-configuration.md

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+   3. Obtain the access key.   
+            
+      <img src="/assets/img/connectors/MS-azure-storage-copy-access-key.png" title="Copy access keys" width="800" alt="Copy access keys"/>   
+           
+> **Note**:  Azure Storage Account does not support HTTP requests. If you're using storage key to access the storage account, please set **Secure transfer required** to **Disabled** in storage account configuration on Azure Portal.  
+    <img src="/assets/img/connectors/MS-azure-storage-http-request.png" title="Copy access keys" width="800" alt="Copy access keys"/> 
+  
+## Setting up the Microsoft Azure Storage Connector
+
+Before you start configuring the Microsoft Azure Storage Connector you also need WSO2 EI and we refer to that location as <PRODUCT_HOME>.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:79]:
```suggestion
Before you start configuring the Microsoft Azure Storage Connector, you also need WSO2 EI and we refer to that location as <PRODUCT_HOME>.
```

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+In order to use the Microsoft Azure Storage connector, you need to download following jar file and **copy** the **jar** to the `<PRODUCT_HOME>/lib directory`.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:81]:
```suggestion
In order to use the Microsoft Azure Storage connector, you need to download the following .jar file and move it to the `<PRODUCT_HOME>/lib directory`.  
```

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+      
+   2. Click **Access keys** under **Settings**.
+   
+      <img src="/assets/img/connectors/MS-azure-storage-getting-accesskeys.png" title="Getting access keys" width="800" alt="Getting access keys"/>
+      
+   3. Obtain the access key.   
+            
+      <img src="/assets/img/connectors/MS-azure-storage-copy-access-key.png" title="Copy access keys" width="800" alt="Copy access keys"/>   
+           
+> **Note**:  Azure Storage Account does not support HTTP requests. If you're using storage key to access the storage account, please set **Secure transfer required** to **Disabled** in storage account configuration on Azure Portal.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:74]:
```suggestion
> **Note**:  Azure Storage Account does not support HTTP requests. If you are using a storage key to access the storage account, please set **Secure transfer required** to **Disabled** in storage account configuration on Azure Portal.  
```

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+      
+   9. Select **Review + Create** to review your storage account settings and create the account.
+   
+      <img src="/assets/img/connectors/MS-azure-storage-review-create.png" title="Review and create" width="800" alt="Review and create"/>
+      
+   10. Select **Create**.
+
+## Obtaining the access credentials   
+   
+   1. Navigate to created **storage account** and click.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:62]: Is there something missing here? Click what?

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+    
+## Create Microsoft Azure Storage account
+
+Follow the steps below to obtain the access credentials from Microsoft Azure Storage account.
+
+   1. Go to [Microsoft Azure](https://azure.microsoft.com/en-in/free/), and sign in to the created Microsoft Azure account. On the Azure portal menu, select **All services**. In the list of resources, type **Storage Accounts**. As you begin typing, the list filters based on your input. Select **Storage Accounts**.                                                                                                                           
+      
+      <img src="/assets/img/connectors/MS-azure-storage-homepage.png" title="MS Azure Home Page" width="800" alt="MS Azure Home Page"/>MS-azure-storage-select-account.png   
+
+   2. Go to the dashboard and click Storage accounts then click add and fill the required details to create new storage account.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:25]:
```suggestion
   2. Go to the dashboard and click **Storage accounts** then click **Add** and fill the required details to create a new storage account.
```

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md (hunk 4 above)
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:62]:
```suggestion
   1. Navigate to the created **storage account** and click it. 
```

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md (hunk 5 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:25]: Done.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md (hunk 4 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:62]: Done.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:74]: Updated.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:79]: Updated.

File: en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration.md:81]: Updated.

**Pull request merged**